tilequeue/command.py

```diff
@@ -577,7 +577,8 @@ def tilequeue_process(cfg, peripherals):
     assert n_simultaneous_query_sets > 0
     # reduce queue size when we're rendering metatiles to try and avoid the
     # geometry waiting to be processed from taking up all the RAM!
-    default_queue_buffer_size = max(1, 128 >> (2 * (cfg.metatile_size or 0)))
+    default_queue_buffer_size = max(
+        1, 128 >> ((2 * (cfg.metatile_size or 0))) + 1)
```

Comment:
This works for what I wanted to do (reduce it by a factor of 2), but probably not the best way to calculate it. Is there a better way to spell it?